sentences/de/person_HassGetState.yaml

```diff
@@ -0,0 +1,121 @@
+language: de
+intents:
+  HassGetState:
+    data:
+      # https://www.home-assistant.io/integrations/person/
+      - sentences:
+          - "<wo_ist> <name>"
+        response: wo
+        requires_context:
+          domain: person
+        slots:
+          domain: person
```

Comment:
i think we do not need to set the `domain` here, because above we already require the domain to be `person` 🤔

Reply:
this should also count for the sentences below 😉

Reply:
after some testing it figured out, that setting `slot.domain` is still needed, even we already require domain to be person 🤔